src/global_mapper/global_mapper.h

```diff
@@ -6,16 +6,22 @@
 #include <vector>
 #include <thread>
 #include <mutex>
+#include <memory>
+#include <condition_variable>
 
 #include "pcl_ros/point_cloud.h"
+#include "global_mapper/global_mapper_params.h"
+
+#include "occ_map/voxel_map.hpp"
+#include "occ_map/pixel_map.hpp"
 
 namespace global_mapper {
 
 typedef pcl::PointCloud<pcl::PointXYZ> PointCloud;
```

Comment:
The problem with putting it inside GlobalMapper is now you have to write GlobalMapper::PointCloud every time you use it in the source file.  By making the typename that long it all but eliminates the purpose of the typedef in the first place.  Until I find myself using other types of pointclouds, I think it is best to define it within the namespace.  Besides, I can always just make it PointCloudXYZ.  That is far better than having to scope with the class name every time.